src/encoded/tests/test_types_donor.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+
+@pytest.fixture
+def parent(testapp, award, lab, human):
```

Comment:
More descriptive name. `parent_donor`?

Reply:
I changed the name to parent_human_donor. 

I included human because donor.py has calculated properties for donors from many organisms, not just human.